src/SerializeAST.cc

```diff
@@ -33,6 +39,28 @@ using snowcrash::Action;
 using snowcrash::Resource;
 using snowcrash::Blueprint;
 
+#ifdef _WITH_REFRACT_
+/**
+ * FIXME:
+ * hotfix until solve ErrorReporting from drafter into snowcrash result.
+ * we need it to allow free allocated memory
+ * this flag WILL BE REMOVED, 
+ * DO NOT USE it until you know what are you doing
+ */
+
+enum {
+    NoError = 0,
+    RuntimeError
+};
+
+
+static int DrafterErrorCode = NoError;
+static std::string DrafterErrorMessage;
+
+static bool ExpandMSON = false;
```

Comment:
This could be made similar to `GetNamedTypesRegistry` but that can be done later